### DISCUSSION TITLE
Settings menu - Always on top behaviour

### DISCUSSION THREAD

**Discussion by chip33** [Hofknecht/SystemTrayMenu#416] [Q&A] [answered]
I noticed that the Settings menu is set to Always on top which means you have to mouse click the minimize button to hide it.
Would it be better if it was not Always on top? 
I checked with some other applications to see what is the more usual behaviour and their Settings menus are not generally set to Always on top.

**Comment by Hofknecht** [comment, marked as answer]
@chip33 Thank you we created issue from discussion
#417
